catalog/ui/catalog-ui-search/src/main/webapp/styles/components/layers.less

```diff
@@ -14,8 +14,10 @@
         width: 100%;
         margin-bottom: 0px;
     }
+
     input[type="checkbox"] {
-        display: none;
+        vertical-align: middle;
+        padding: 0px;
```

Comment:
I made the change, but the checkboxes are still slightly different between chrome and ff.